### ISSUE TITLE
no imports/ folder ?

### ISSUE THREAD

**Issue by sahanDissanayake** [kadira-samples/meteor-data-and-react#8]
http://guide.meteor.com/1.3-migration.html#modules

**Comment by sahanDissanayake** [comment]
@arunoda I made some minor changes to this repo to get the code to work with imports/ folder
https://github.com/sahanDissanayake/es2015-ssr

**Comment by matthlavacka** [comment]
Looking exactly for this! Thanks

**Comment by mhlavacka** [comment]
Use `meteor npm i --save` instead of just `npm i --save` because now it saves all your npm packages to node_modules.

**Comment by sascha1337** [comment]
Yes this is what i want to see sahan great work. 

What about Material UI / Bootstrap versions with Facebook Auth on SSR ? Not seen yet.

**Comment by sahanDissanayake** [comment]
are you using my structure @mhlavacka , @sascha1337  ? if so is everything working alright ?

**Comment by matthlavacka** [comment]
@sahanDissanayake Yes, I made some little changes, but everything works alright. thanks mate!

**Comment by sahanDissanayake** [comment]
Sweet. Good to hear.. the changes you made were not bugs was it ? thanks

**Comment by matthlavacka** [comment]
No, just minor changes in the structure based on [Simple todos react](https://github.com/meteor/simple-todos-react/)
